doc/developers/contributing.rst

```diff
@@ -971,13 +971,16 @@ To also generate the example gallery you can use:
 
     make html
 
-This will run all the examples, which takes a while. If you only want to generate a few
-examples, which is particularly useful if you are modifying only a few examples, you can
-use:
+This will run all the examples, which takes a while. If you only want to generate 
+images for a few examples, you can pass their names (or parts of) to the build command.
+The line below will run all examples with names starting with `plot_calibration`:
```

Comment:
```suggestion
This will run all the examples, which takes a while. You can also run only a few examples based on their file names.
Here is a way to run all examples with file names containing with `plot_calibration`:
```